GooglePlacesAutocomplete.js

```diff
@@ -116,6 +116,22 @@ export default class GooglePlacesAutocomplete extends Component {
     return [...res, ...results];
   }
 
+
+  addAnnotations = (firstObj, annotations) => {
+    let newRes = this.state.dataSource;
+    // don't have to worry about locking and multithreading
+    // but make sure we are updating the right list
+    if (annotations.length > 0 && firstObj == newRes[0].description) {
```

Comment:
do we want a triple equals?
```suggestion
    if (annotations.length > 0 && firstObj === newRes[0].description) {
```

Reply:
yup makes sense